packages/lib/shared/hooks/useCurrency.ts

```diff
@@ -58,5 +58,13 @@ export function useCurrency() {
     return withSymbol ? symbol + formattedAmount : formattedAmount
   }
 
-  return { toCurrency, formatCurrency, parseCurrency }
+  // Formats currency balance with en-dash for zero values
+  function formatCurrencyBalance(usdVal: Numberish, options: CurrencyOpts = {}): string {
+    if (isZero(usdVal)) {
+      return ZERO_VALUE_DASH
+    }
+    return toCurrency(usdVal, options)
+  }
```

Comment:
if im not mistaken this looks unused? ( try a `ctrl + shift + f` for `formatCurrencyBalance` )

Reply:
removed, thx